### PR TITLE
Convert BitBlt to cswin32 PInvoke

### PR DESCRIPTION
**Pull request by lonitra**
- Add HDC conversion to scopes
- Remove unused BeginPath

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/7446)

### CODE REVIEW

**Review by JeremyKuhne — APPROVED** [review]
Looks good! Welcome back!!

File: src/System.Windows.Forms.Primitives/src/NativeMethods.txt [hunk 1]
```diff
@@ -8,3 +8,4 @@ GlobalUnlock
 GetCurrentProcessId
 GlobalReAlloc
 GlobalFree
+BitBlt
```
Comment (RussKie) [inline src/System.Windows.Forms.Primitives/src/NativeMethods.txt:11]: Similar to https://github.com/dotnet/winforms/pull/7444#discussion_r923987009 - please have this file sorted in alphabetical order.

File: src/System.Windows.Forms.Primitives/src/Windows/Win32/PInvoke.BitBlt.cs [hunk 2]
```diff
@@ -2,34 +2,23 @@
...
-            ROP rop)
+               IHandle hdc,
+               int x,
+               int y,
+               int cx,
+               int cy,
+               HDC hdcSrc,
+               int x1,
+               int y1,
+               ROP_CODE rop)
```
Comment (RussKie) [inline src/System.Windows.Forms.Primitives/src/Windows/Win32/PInvoke.BitBlt.cs:21]: Why did the indentation change?

File: src/System.Windows.Forms/src/System/Windows/Forms/ControlPaint.cs [hunk 3]
```diff
@@ -9,7 +9,9 @@
 using System.Drawing.Drawing2D;
 using System.Drawing.Imaging;
 using System.Runtime.InteropServices;
+using Windows.Win32;
 using static Interop;
+using Gdi = Windows.Win32.Graphics.Gdi;
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/ControlPaint.cs:14]: [nit] Are these sorted? If not, please have these sorted (via the context menu):
image

Ditto below

**Review by RussKie — COMMENTED** [review]
👍 with stylistics nits

**Review by AArnott — APPROVED** [review]

File: src/System.Windows.Forms.Primitives/src/Windows/Win32/PInvoke.BitBlt.cs (hunk 2 above)
Comment (lonitra) [reply src/System.Windows.Forms.Primitives/src/Windows/Win32/PInvoke.BitBlt.cs]: Oops! Must've happened by accident when I was moving things around.

**Review by Tanya-Solyanik — APPROVED** [review]
Welcome back!

**Pull request merged**